src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int UNITS = 1000;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        String color = colorSupplier.getRandomColor();
+        switch (figureNumber) {
+            case 1:
+                return square(color);
+            case 2:
+                return rectangle(color);
+            case 3:
+                return circle(color);
+            case 4:
+                return rightTriangle(color);
+            default:
+                return isoscelesTrapezoid(color);
+        }
+    }
+
+    private Figure square(String color) {

```

Comment:
```suggestion
    private Figure getSquare(String color) {

```

названия методов должны начинаться с глагола. измени во всех следующих методах